packages/astro-rss/test/rss.test.js

```diff
@@ -44,10 +46,28 @@ describe('rss', () => {
 		});
 
 		const str = await response.text();
-		chai.expect(str).xml.to.equal(validXmlResult);
+
+		// NOTE: Chai used the below parser to perform the tests, but I have omitted it for now.
+		// parser = new xml2js.Parser({ trim: flag(this, 'deep') });
+
+		// NOTE: The below test code also works, if we do not want to add the parseXmlString() function to test-utils.js
+		// I have already added the function, which returns an object with signature of: { err, result }.
+		// If there is an error, the test will fail (no err present in the expected result)
+
+		/*
+		parseXmlString(str, (err, actual) => {
+			assert.strictEqual(err, null);
+			parseXmlString(validXmlResult, (err, predicted) => {
+				assert.strictEqual(err, null);
+				assert.deepStrictEqual(actual, predicted);
+			});
+		});
+		*/
```

Comment:
Comment is here, let me know what you think about putting the function and import in `test-utils.js`! I think it works well, but I'm not sure :)

Reply:
I think it works well too! A suggestion: the function should check for errors, and if there are any, it should throw before asserting that the two results are the same. Also, we should assert the equality only of the results, not the errors too.

Reply:
> I think it works well too! A suggestion: the function should check for errors, and if there are any, it should throw before asserting that the two results are the same. Also, we should assert the equality only of the results, not the errors too.

Alright, can do! If you have any input regarding what it should throw, I would appreciate it! Otherwise, I will come up with something :)